jekyll/_cci2/introduction-to-yaml-configurations.adoc

```diff
@@ -0,0 +1,240 @@
+
+The developer hub also has a list of available CircleCI https://circleci.com/developer/orbs[orbs], which are shareable packages of configuration you can use to simplify your builds. Visit the <<slack-orb-tutorial#,Slack Orb Tutorial>> page for instructions on how to set up one of the most popular orbs.
+
+[#circleci-jobs]
+== CircleCI jobs and workflows
+
+The last thing we need for a basic CircleCI configuration is a job to actually run. Jobs are orchestrated using workflows, which is a set of rules defining a collection of jobs and their run order. Using the Docker configuration example, we can add steps to our job. Steps are a list of commands to run inside the Docker container.
+
+```yaml
+# CircleCI configuration file
+version: 2.1
+
+jobs:
+  build:
+    docker:
+     - image: cimg/base:2022.05
+    steps:
+        - run: echo "Say hello to YAML!"
+```
+
+This very basic example is only running one job, `build`, which means there is also only one workflow running in the background. If a second job is added, `workflows` will need to be defined explicitly, to orchestrate the run order. Job names can be updated to reflect the job when `workflows` is added. If there is only one job defined, the job must be named `build`, as in the example above.
+
+A second job, and `workflows` defining run order, is shown in the example below.
+
+```yaml
+# CircleCI configuration file
+version: 2.1
+
+jobs:
+  # Job one with a unique name
+  say_hello:
+    docker:
+     - image: cimg/base:2022.05
+    steps:
+      - run: echo "Say hello to YAML!"
+  # Job two with a unique name
+  say_goodbye:
+    docker:
+     - image: cimg/base:2022.05
+    steps:
+      - run: echo "Say goodbye to YAML!"
+
+workflows:
+  # Name of workflow
+  hello_and_goodbye:
+    # List of jobs that will run
+    jobs:
+      - say_hello
+      - say_goodbye
+```
+
+If you have a CircleCI account, you can create a new project and add these examples to a `.circleci/config.yaml` file. You can see the strings printed out in the job's build pipeline in the CircleCI web UI.
+
+YAML can be quite picky about indentations. You can use a http://yaml-online-parser.appspot.com/[YAML checker] to parse your YAML and make sure it is valid.
+
+If you would like a more complex configuration tutorial, visit the <<config-intro#,Configuration Tutorial>> page. Before starting this tutorial, you should already have a CircleCI account set up, as you will follow along in the CircleCI web UI. You can also find a variety of configuration examples on the <<sample-config#, Sample Configuration>> page.
+
+[#fun-with-yaml]
+== Fun with YAML
+
+Below are some fun examples of other YAML syntax that might become handy as you create more complex configuration files.
+
+[#multi-line-strings]
+=== Multi-line strings
+
+If the value is a multi-line string, use the `>` character, followed by any number of lines. This is especially useful for lengthy commands.
+
+```yaml
+haiku: >
```

Comment:
I'm not super familiar with syntax specifics but is this another example of multiline strings? Or is the pipe character for an entirely different purpose..? https://github.com/ryanpedersen42/circleci-react-test-splitting/blob/master/.circleci/config.yml#L18